src/javascript/app/base/header.js

```diff
@@ -228,11 +230,26 @@ const Header = (() => {
         });
     };
 
+    const showGoToDerivAlertPopup = () => {
+        const url_on_confirm = `https://app.deriv.com/?lang=${getLanguage()}`;
+        Dialog.confirm({
+            id               : 'go-to-deriv-popup',
+            localized_title  : localize('Go to Deriv to add an account'),
+            localized_message: localize('You\'ll be able to log in to Deriv using your Binary.com credentials.'),
+            cancel_text      : localize('Cancel'),
+            ok_text          : localize('Go to Deriv'),
+            onConfirm        : () => { window.location.href = url_on_confirm; },
+            onAbort          : undefined,
+        });
+    };
+
     const showHidePulser = (should_show) => { $('.upgrademessage').children('a').setVisibility(should_show); };
 
     const showHideNewAccount = (upgrade_info) => {
+        const user_accounts = getElementById('user_accounts');
         if (upgrade_info.can_upgrade || upgrade_info.can_open_multi) {
```

Comment:
```suggestion
        if ((upgrade_info.can_upgrade || upgrade_info.can_open_multi) && user_accounts) {
```
Best to add a check to make sure there is an element that matches the id `user_accounts` before adding the event